deployment/ansible/group_vars/all

```diff
@@ -3,7 +3,9 @@ app_username: "vagrant"
 
 packer_version: "0.7.5"
 
-nodejs_version: 4.6.0
+nodejs_version: 4.6.2
+
+virtualenv_version: 15.1.0
```

Comment:
I'm not sure now if that one was necessary. In any case, the default version in upstream is in need of an update. How about I create an issue there, and an issue here to remove this with role update?